tests/PHPUnit/System/expected/test_OneVisitorTwoVisits__DevicesDetection.getType_day.xml

```diff
@@ -37,6 +37,12 @@
 		<logo>plugins/DevicesDetection/images/screens/mobile.gif</logo>
 	</row>
 	<row>
+		<label>Phablet</label>
+		<nb_visits>0</nb_visits>
+		<segment>deviceType==phablet</segment>
+		<logo>plugins/DevicesDetection/images/screens/unknown.gif</logo>
```

Comment:
@mattab there are also other icons for device types missing. Imho it would be cool to have icons, that fit in the theme and that share the same look-alike. Do you think we could get nice icons for all device types from a designer?